README.md

```diff
@@ -53,12 +59,13 @@ Upload Lambda to AWS and _star_ this repository if it works as expected!!
 
 ### Parameters
 
-| Param  | Description |
-| ------------- | ------------- |
-| event | Event object generated by the stream (pass it as it is)  |
-| index  | The name of ElasticSearch index  |
-| type  | The type of the ElasticSearch document   |
-| endpoint  | Exact url of ElasticSearch instance (it works with AWS ES and standard ES)  |
+| Param  | Description | Options |
+| ------------- | ------------- | ------------- |
+| event | Event object generated by the stream (pass it as it is)  | Parse on from the event handler
```

Comment:
Don't understand the sentence: `Parse on from the event handler`

Maybe, it would be better to write: `Passed by AWS`.
Please add here that it is required and maybe change from (pass it as it is) to (don't modify).